### PR TITLE
VACMS 4926: Enables memcache support on Lando.

### PR DESCRIPTION
**Pull request by ndouglas**
## Description

See #4926.  In accordance with updated ACs, this is enabled by default.  Opening a PR early to assist in work/debugging elsewhere, but it shouldn't be merged yet.  I assume (but haven't tested) that the config:import and possible composer installs will fail if the `memcache` module isn't in place at the time.

As far as I can tell, checking whether the `memcache` extension is loaded in `settings.lando.php` is sufficient for determining whether the cache should be in DB or memcache, so the `memcache-on` and `memcache-off` Lando commands can be fairly simple.  I threw a non-skippable cache-rebuild in there.  It might be worth throwing that into a bash script and adding an argument to allow skipping it, but given that this is a development environment I think the possible issues from _not_ rebuilding the cache after changing the location of the cache make this a reasonable decision.

To Do:
- [x] set up for locks
- [x] set up session management (this seems to not need to be handled explicitly because Drupal uses Symfony's session management, unless I'm missing something)
- [x] make sure installation works unattended (currently prompts for session support)

## Testing done
Rebuilding and clicking around.

## Screenshots
With the `memcache` module enabled (by `lando memcache-on`), indicating that information is being read from and written to memcache:
Screen Shot 2021-04-12 at 4 40 52 PM

**EDIT**: Originally included the wrong screenshot 😩 

With the `memcache` module disabled (by `lando memcache-off`), indicating that the site continues to function and that information is _not_ being read from or written to memcache:
Screen Shot 2021-04-12 at 4 42 17 PM

## QA steps

As user _uid_ with _user_role_
1. Do this
1. Then that
1. Then validate Acceptance Criteria from issue
- [ ] This
- [ ] That
- [ ] The other thing

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [x] `Core Application Team`
- [ ] `Product Support Team`

### Is this PR blocked by another PR?

- [x] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [x] No announcement is needed for this code change. 
  - [x] Merge & carry on unburdened by announcements

### CODE REVIEW

File: .lando.yml [hunk 1]
```diff
@@ -36,6 +36,7 @@ services:
       # cypress-axe dependencies - https://docs.cypress.io/guides/getting-started/installing-cypress.html#System-requirements
       - "apt-get install libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb -y"
       - .lando/scripts/blackfire-init.sh
+      - pecl install memcache-4.0.5.2 && docker-php-ext-enable memcache
```
Comment (indytechcook) [inline .lando.yml:39]: This shouldn't be needed

Comment (ndouglas) [reply .lando.yml:39]: The `memcache` module needs to be installed.

Comment (indytechcook) [reply .lando.yml:39]: I had it working on lando without it.

Comment (ndouglas) [reply .lando.yml:39]: Screen Shot 2021-04-14 at 9 20 56 AM

Comment (ndouglas) [reply .lando.yml:39]: Were you perhaps using `memcached`?

Comment (indytechcook) [reply .lando.yml:39]: Anything is possible.

**Comment by indytechcook** [comment]
Testing this local.  After a `lando composer nuke` and a `lando rebuild` and a `./scripts/sync_db.sh`:

image

Ran `lando memcache-off` and the cache was still enabled

image

I also monitored the number of commands that ran and and they did increase when memcache was supposed to be not be enabled.

**Comment by ndouglas** [comment]
D'oh, it just falls back to using `memcached`, which Lando installs by default (as mentioned above) 🤦  I added a line to remove the `memcached` configuration and reload Apache, and now get appropriate errors.

**Review by indytechcook — APPROVED** [review]

**Pull request merged**